content/articles/building-a-simple-translation-app-using-python-for-beginners/index.md

```diff
@@ -0,0 +1,171 @@
+
+Pipelines are a great and easy way to use models for inference. Hugging Face has many pipelines including the `TokenClassificationPipeline`, `ZeroShotClassificationPipeline` and `Text2TextGenerationPipeline` just to name a few. In our tutorial, we'll be using the `TranslationPipeline`. This pipeline has been fine-tuned to perform translation tasks.
+
+Technology companies such as [Speech Brain](https://speechbrain.github.io/) and [GoogleAI](https://ai.google/research/) use the hugging face model to perform deep learning and create speech technologies.     
+
+[Gradio](https://gradio.app/), on the other hand, allows us to quickly create customizable UI components around PyTorch models. It allows you to create interfaces for your ML models. It helps you quickly test the models you've created using a web interface. There are free example code implementations on their official website demonstrating how you can use gradio to perform digit classification, face segmentation, and outbreak forecast. Please feel free to check it out.
+
+### Installing dependencies
+> An important point to note before installing PyTorch is that Google Colab comes pre-installed with PyTorch. You may need to ignore the PyTorch installation if you're using Colab and skip to the Hugging Face and Gradio installations. Only follow the steps to install PyTorch if you're installing it on your machine.
+
+Our main dependencies are going to be PyTorch, Hugging Face Transformers, and Gradio. Let's go ahead and install them.
+
+Let's first install PyTorch. This is available [here](https://pytorch.org/) on PyTorch's website. On the website, find the `Install` button and press it. On the install page, you will see a walkthrough section similar to the one shown below:
+
+![Installing PyTorch](/engineering-education/building-a-simple-translation-app-using-python-for-beginners/installing-pytorch.png)
+
+*[Image Source: PyTorch](https://pytorch.org/)*
+
+The commands for installing PyTorch vary depending on factors such as your OS, package of choice (pip, conda, source), language, and compute platform (CUDA, CPU, or ROCm). So, make sure you select your preference. It doesn't have to look like mine. Mine's shown below:
+
+![Preference](/engineering-education/building-a-simple-translation-app-using-python-for-beginners/pytorch-preference.png)
+
+*[Image Source: PyTorch](https://pytorch.org/)*
+
+To install PyTorch on my machine, I've selected the Long Term Support (LTS 1.8.1) build, Linux OS, `pip` package, Python programming language, and CPU as my compute platform of choice. If your machine has GPU support, choose accordingly.  
+
+After selecting those options, this is the command you're going to run to install PyTorch:
+
+```python
+    pip3 install torch==1.8.1+cpu torchvision==0.9.1+cpu torchaudio==0.8.1 -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html
```

Comment:
The code has unnecessary indentations (the initial spaces). Please remove them in all the code blocks in the draft.